src/Drag.js

```diff
@@ -462,6 +462,8 @@ class Drag extends State {
 	 * @protected
 	 */
 	handleDragStartEvent_(event) {
+		event.stopPropagation();
```

Comment:
I see that `preventDefault` is also called in this method, but only when `canStartDrag_` returns true. Can we do that same for `stopPropagation`?

Reply:
It will cause overwriting of this.activeDragSource_ and it is exactly what we want to avoid.

Reply:
Maybe we can move the `activeDragSource_` inside the if as well?